.github/workflows/test.yml

```diff
@@ -10,6 +10,8 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 15
+    env:
+      PYTHON_COVERAGE_ENABLED: '3.8'
```

Comment:
Happy to change (or if anyone commits to this branch changing this value)

Reply:
Seems like a sensible value to me.